From b318c692a269e201eca2def67ad0eb1724442e16 Mon Sep 17 00:00:00 2001
From: Daniel Goehring <dgoehrin@os.amperecomputing.com>
Date: Thu, 16 Apr 2020 15:43:37 -0400
Subject: [PATCH 28/33] QS|MQ: emulation user configs [DO NOT UPSTREAM]

Add user configuration files for Ampere Altra
and Ampere Altra Max in Emulation.

Protium configuration uses an Olimex ARM-USB-OCD_H JTAG probe.
https://www.olimex.com/Products/ARM/JTAG/ARM-USB-OCD-H/

OpenOCD configured for the following QS options
1. JTAG DPI Emulation interface (Palladium)
2. JTAG Olimex (Protium)
3. One and two socket platforms

Tested on the following system platforms
* Palladium 1S2C1M and 2S2C1M
* Protium

Change-Id: Ia8d5b29418557c947cb603a2070bc927152ee415
Signed-off-by: Daniel Goehring <dgoehrin@os.amperecomputing.com>
---
 tcl/openocd.mq_1s_pal.cfg  | 69 ++++++++++++++++++++++++++++++++
 tcl/openocd.mq_1s_prot.cfg | 74 ++++++++++++++++++++++++++++++++++
 tcl/openocd.mq_2s_pal.cfg  | 77 +++++++++++++++++++++++++++++++++++
 tcl/openocd.mq_2s_prot.cfg | 82 ++++++++++++++++++++++++++++++++++++++
 tcl/openocd.qs_1s_pal.cfg  | 68 +++++++++++++++++++++++++++++++
 tcl/openocd.qs_1s_prot.cfg | 73 +++++++++++++++++++++++++++++++++
 tcl/openocd.qs_2s_pal.cfg  | 76 +++++++++++++++++++++++++++++++++++
 tcl/openocd.qs_2s_prot.cfg | 81 +++++++++++++++++++++++++++++++++++++
 8 files changed, 600 insertions(+)
 create mode 100644 tcl/openocd.mq_1s_pal.cfg
 create mode 100644 tcl/openocd.mq_1s_prot.cfg
 create mode 100644 tcl/openocd.mq_2s_pal.cfg
 create mode 100644 tcl/openocd.mq_2s_prot.cfg
 create mode 100644 tcl/openocd.qs_1s_pal.cfg
 create mode 100644 tcl/openocd.qs_1s_prot.cfg
 create mode 100644 tcl/openocd.qs_2s_pal.cfg
 create mode 100644 tcl/openocd.qs_2s_prot.cfg

diff --git a/tcl/openocd.mq_1s_pal.cfg b/tcl/openocd.mq_1s_pal.cfg
new file mode 100644
index 000000000..21ce60d90
--- /dev/null
+++ b/tcl/openocd.mq_1s_pal.cfg
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra Max
+# Development Platform
+#
+# Copyright (c) 2020-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# DPI_PORT
+# Set the DPI JTAG server port assigned to the emulator BFM transactor
+# Syntax: -c "set DPI_PORT {5555}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "mq"
+# Syntax: -c "set SYSNAME {mq}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set CORELIST_S0 {16 17}"
+#
+# COREMASK_S0_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 94 and 95 for S0
+# Syntax: -c "set COREMASK_S0_HI {0x00000000C0000000}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/jtag_dpi.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME mq
+}
+
+set MQ_ENABLE ""
+source [find board/ampere_qs_mq_1s.cfg]
diff --git a/tcl/openocd.mq_1s_prot.cfg b/tcl/openocd.mq_1s_prot.cfg
new file mode 100644
index 000000000..a5109db2f
--- /dev/null
+++ b/tcl/openocd.mq_1s_prot.cfg
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra Max
+# Development Platform
+#
+# Copyright (c) 2020-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# JTAGFREQ
+# Override the default JTAG clock frequency
+# If not specified, defaults to "100" kHz
+# Syntax: -c "set JTAGFREQ {100}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "mq"
+# Syntax: -c "set SYSNAME {mq}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set CORELIST_S0 {16 17}"
+#
+# COREMASK_S0_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 94 and 95 for S0
+# Syntax: -c "set COREMASK_S0_HI {0x00000000C0000000}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/ftdi/olimex-arm-usb-ocd-h.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists JTAGFREQ] } {
+	set JTAGFREQ 100
+}
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME mq
+}
+
+set MQ_ENABLE ""
+source [find board/ampere_qs_mq_1s.cfg]
diff --git a/tcl/openocd.mq_2s_pal.cfg b/tcl/openocd.mq_2s_pal.cfg
new file mode 100644
index 000000000..0496b7d3a
--- /dev/null
+++ b/tcl/openocd.mq_2s_pal.cfg
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra Max
+# Development Platform
+#
+# Copyright (c) 2020-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# DPI_PORT
+# Set the DPI JTAG server port assigned to the emulator BFM transactor
+# Syntax: -c "set DPI_PORT {5555}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "mq"
+# Syntax: -c "set SYSNAME {mq}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0, CORELIST_S1
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set CORELIST_S0 {16 17}"
+# Syntax: -c "set CORELIST_S1 {16 17}"
+#
+# COREMASK_S0_LO, COREMASK_S1_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+# Syntax: -c "set COREMASK_S1_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI, COREMASK_S1_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 94 and 95 for S0 and S1
+# Syntax: -c "set COREMASK_S0_HI {0x00000000C0000000}"
+# Syntax: -c "set COREMASK_S1_HI {0x00000000C0000000}"
+#
+# SPLITSMP
+# Group all ARMv8 cores per socket into individual SMP sessions
+# If not specified, group ARMv8 cores from both sockets into one SMP session
+# Syntax: -c "set SPLITSMP {}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/jtag_dpi.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME mq
+}
+
+set MQ_ENABLE ""
+source [find board/ampere_qs_mq_2s.cfg]
diff --git a/tcl/openocd.mq_2s_prot.cfg b/tcl/openocd.mq_2s_prot.cfg
new file mode 100644
index 000000000..c33cb1bcc
--- /dev/null
+++ b/tcl/openocd.mq_2s_prot.cfg
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra Max
+# Development Platform
+#
+# Copyright (c) 2020-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# JTAGFREQ
+# Override the default JTAG clock frequency
+# If not specified, defaults to "100" kHz
+# Syntax: -c "set JTAGFREQ {100}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "mq"
+# Syntax: -c "set SYSNAME {mq}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0, CORELIST_S1
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set CORELIST_S0 {16 17}"
+# Syntax: -c "set CORELIST_S1 {16 17}"
+#
+# COREMASK_S0_LO, COREMASK_S1_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+# Syntax: -c "set COREMASK_S1_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI, COREMASK_S1_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 94 and 95 for S0 and S1
+# Syntax: -c "set COREMASK_S0_HI {0x00000000C0000000}"
+# Syntax: -c "set COREMASK_S1_HI {0x00000000C0000000}"
+#
+# SPLITSMP
+# Group all ARMv8 cores per socket into individual SMP sessions
+# If not specified, group ARMv8 cores from both sockets into one SMP session
+# Syntax: -c "set SPLITSMP {}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/ftdi/olimex-arm-usb-ocd-h.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists JTAGFREQ] } {
+	set JTAGFREQ 100
+}
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME mq
+}
+
+set MQ_ENABLE ""
+source [find board/ampere_qs_mq_2s.cfg]
diff --git a/tcl/openocd.qs_1s_pal.cfg b/tcl/openocd.qs_1s_pal.cfg
new file mode 100644
index 000000000..349b82ee3
--- /dev/null
+++ b/tcl/openocd.qs_1s_pal.cfg
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra
+# Development Platform
+#
+# Copyright (c) 2019-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# DPI_PORT
+# Set the DPI JTAG server port assigned to the emulator BFM transactor
+# Syntax: -c "set DPI_PORT {5555}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "qs"
+# Syntax: -c "set SYSNAME {qs}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set CORELIST_S0 {16 17}"
+#
+# COREMASK_S0_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 78 and 79 for S0
+# Syntax: -c "set COREMASK_S0_HI {0x000000000000C000}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/jtag_dpi.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME qs
+}
+
+source [find board/ampere_qs_mq_1s.cfg]
diff --git a/tcl/openocd.qs_1s_prot.cfg b/tcl/openocd.qs_1s_prot.cfg
new file mode 100644
index 000000000..514b464eb
--- /dev/null
+++ b/tcl/openocd.qs_1s_prot.cfg
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra
+# Development Platform
+#
+# Copyright (c) 2019-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# JTAGFREQ
+# Override the default JTAG clock frequency
+# If not specified, defaults to "100" kHz
+# Syntax: -c "set JTAGFREQ {100}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "qs"
+# Syntax: -c "set SYSNAME {qs}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set CORELIST_S0 {16 17}"
+#
+# COREMASK_S0_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 78 and 79 for S0
+# Syntax: -c "set COREMASK_S0_HI {0x000000000000C000}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/ftdi/olimex-arm-usb-ocd-h.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists JTAGFREQ] } {
+	set JTAGFREQ 100
+}
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME qs
+}
+
+source [find board/ampere_qs_mq_1s.cfg]
diff --git a/tcl/openocd.qs_2s_pal.cfg b/tcl/openocd.qs_2s_pal.cfg
new file mode 100644
index 000000000..ea1a79562
--- /dev/null
+++ b/tcl/openocd.qs_2s_pal.cfg
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra
+# Development Platform
+#
+# Copyright (c) 2019-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# DPI_PORT
+# Set the DPI JTAG server port assigned to the emulator BFM transactor
+# Syntax: -c "set DPI_PORT {5555}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "qs"
+# Syntax: -c "set SYSNAME {qs}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0, CORELIST_S1
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set CORELIST_S0 {16 17}"
+# Syntax: -c "set CORELIST_S1 {16 17}"
+#
+# COREMASK_S0_LO, COREMASK_S1_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+# Syntax: -c "set COREMASK_S1_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI, COREMASK_S1_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 78 and 79 for S0 and S1
+# Syntax: -c "set COREMASK_S0_HI {0x000000000000C000}"
+# Syntax: -c "set COREMASK_S1_HI {0x000000000000C000}"
+#
+# SPLITSMP
+# Group all ARMv8 cores per socket into individual SMP sessions
+# If not specified, group ARMv8 cores from both sockets into one SMP session
+# Syntax: -c "set SPLITSMP {}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/jtag_dpi.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME qs
+}
+
+source [find board/ampere_qs_mq_2s.cfg]
diff --git a/tcl/openocd.qs_2s_prot.cfg b/tcl/openocd.qs_2s_prot.cfg
new file mode 100644
index 000000000..e5afcd1ea
--- /dev/null
+++ b/tcl/openocd.qs_2s_prot.cfg
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Example OpenOCD user configuration for the Ampere Altra
+# Development Platform
+#
+# Copyright (c) 2019-2023, Ampere Computing LLC
+#
+
+# Command Line Argument Description
+#
+# JTAGFREQ
+# Override the default JTAG clock frequency
+# If not specified, defaults to "100" kHz
+# Syntax: -c "set JTAGFREQ {100}"
+#
+# SYSNAME
+# Override the default system name
+# If not specified, defaults to "qs"
+# Syntax: -c "set SYSNAME {qs}"
+#
+# Life-Cycle State (LCS)
+# If not specified, defaults to "Secure LCS"
+# LCS=0, "Secure LCS"
+# LCS=1, "Chip Manufacturing LCS"
+# Syntax: -c "set LCS {0}"
+# Syntax: -c "set LCS {1}"
+#
+# CORELIST_S0, CORELIST_S1
+# Specify available physical cores by number
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set CORELIST_S0 {16 17}"
+# Syntax: -c "set CORELIST_S1 {16 17}"
+#
+# COREMASK_S0_LO, COREMASK_S1_LO
+# Specify available physical cores 0-63 by mask
+# Example syntax to connect to physical cores 16 and 17 for S0 and S1
+# Syntax: -c "set COREMASK_S0_LO {0x0000000000030000}"
+# Syntax: -c "set COREMASK_S1_LO {0x0000000000030000}"
+#
+# COREMASK_S0_HI, COREMASK_S1_HI
+# Specify available physical cores 64 and above by mask
+# Example syntax to connect to physical cores 78 and 79 for S0 and S1
+# Syntax: -c "set COREMASK_S0_HI {0x000000000000C000}"
+# Syntax: -c "set COREMASK_S1_HI {0x000000000000C000}"
+#
+# SPLITSMP
+# Group all ARMv8 cores per socket into individual SMP sessions
+# If not specified, group ARMv8 cores from both sockets into one SMP session
+# Syntax: -c "set SPLITSMP {}"
+#
+# PHYS_IDX
+# Enable OpenOCD ARMv8 core target physical indexing
+# If not specified, defaults to OpenOCD ARMv8 core target logical indexing
+# Syntax: -c "set PHYS_IDX {}"
+
+bindto 0.0.0.0
+#debug_level 3
+
+#gdb_port 3333
+#telnet_port 4444
+#tcl_port 6666
+
+#
+# Interface Configuration
+#
+
+source [find interface/ftdi/olimex-arm-usb-ocd-h.cfg]
+
+#
+# Board (Platform) Configuration
+#
+
+if { ![info exists JTAGFREQ] } {
+	set JTAGFREQ 100
+}
+
+if { ![info exists SYSNAME] } {
+	set SYSNAME qs
+}
+
+source [find board/ampere_qs_mq_2s.cfg]
-- 
2.25.1

